### DISCUSSION TITLE
No GNU/Linux distro packages/repositories available

### DISCUSSION THREAD

**Discussion by jschwartzenberg** [open-watcom/open-watcom-v2#254] [Ideas]
It would be nice if it would be possible to install from a PPA (Ubuntu) or COPR (Fedora) repository or at least have packages which would integrate with the system.

I was looking into building it myself from source, but I can't seem to get the rel directory to be created which is specified in the readme.

**Comment by jmalak** [comment]
If anybody create some script for creating these packages It can be included into OW tree. But each Linux version will probably require different Linux distribution for creating these packages. It must be somehow handled.

The standard way to create OW distribution is several manual steps or you can use automated way by OW builder which create appropriate scripts for each steps. This run also all tests for new build OW version before creating OW installers.
Details are in Wiki [Build](https://github.com/open-watcom/open-watcom-v2/wiki/Build), See description for builder utility options. For creating/fill rel tree you can use command builder rel.
